build-system/global-configs/prod-config.json

```diff
@@ -3,6 +3,7 @@
     "amp-next-page",
     "disable-amp-story-desktop",
     "inabox-viewport-friendly",
+    "inabox-css-cleanup",
```

Comment:
Why do we need a doc-level opt-in for this?

Reply:
this is an inabox experiment, which can be controlled in ad server